package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "grunt pre-mocha && node test-runner",
-    "prepare": "bower update && grunt init && grunt deploy-build",
+    "prepare": "bower update && grunt deploy-build",
```

Comment:
Calling `deploy-build` slows down the NPM install cycle and should probably be removed. Didn't do that here as I think that something separate to discuss in it's own PR since it would affect some of the CI flow.